### PR TITLE
pygments style plugin

### PR DESCRIPTION
**Pull request by sduff**
pygments plugin, autogenerated by converting the vim colorscheme to the pygments style file.
Copy this file to the pygments/styles directory and then select the lucario formatter as in the following

`pygmentize -S lucario`

### CODE REVIEW

**Comment by raphamorim** [comment]
@sduff can you add guide on README  and a image of using it?

**Comment by sduff** [comment]
I've updated the readme.md to include install instructions
Its a bit difficult to take a screenshot of it, as it's used in a command line to highlight sourcecode (usually for the web)

File: readme.md [hunk 1]
```diff
@@ -121,6 +123,13 @@ $ cp Lucario.dvtcolortheme ~/Library/Developer/Xcode/UserData/FontAndColorThemes
 2.  Copy the **xterm/.Xresources** file to your home directory
 3.  (optional for some cases) Add `xrdb -merge ~/.Xresources` to your init scripts (e.g. `.xinitrc`)
 
+## Pygments
+
+1.  Use `git clone https://github.com/raphamorim/lucario.git` (or use [.zip download][zip] option)
+2.  Copy the **pygments/lucario.py** file to pygments style directory, usually **/usr/local/lib/python/*X*.*X*/site-packages/pygments/styles/**
+3.  Run the following to use the formatter on the command line, `pygmentize -S luciario`
```
Comment (raphamorim) [inline readme.md]: fix it to `pygmentize -S lucario`

File: pygments/lucario.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    ~~~~~~~~~~~~~~~~~~~
+
+		Simon Duff <simon.duff@gmail.com>
+
+"""
+from pygments.style import Style
+from pygments.token import Token, Comment, Name, Keyword, Generic, Number, Operator, String, Error, Literal, Other, Punctuation, Text
+
+class LucarioStyle(Style):
+    default_style = "#f8f8f2"
```
Comment (raphamorim) [inline pygments/lucario.py]: if you're using single quotes for every string, I think that would be good to keep this pattern.

File: pygments/lucario.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    background_color = '#2b3e50'
+
+    styles = {
+			Text: '#f8f8f2',
```
Comment (raphamorim) [inline pygments/lucario.py]: follow the past indentation rule, e.g:
```py
styles = {
    Text: ...
```

**Comment by raphamorim** [comment]
Hi @sduff thank you for contributing with Lucario 👍 
I've made some comments about the your latests changes :D
I'm here for any question.

**Comment by sduff** [comment]
Updated as per your recomendations

**Comment by raphamorim** [comment]
Nice @sduff 
One last little thing: Can you add a sample example image?

**Comment by raphamorim** [comment]
Merged in 045845af72ae57a5eedf3092ceac6652ef39b54c

**Pull request closed without merging**